### PR TITLE
Unify generators into a single program

### PR DESCRIPTION
**Pull request by john-floren-gravwell**
This gathers all the generator types into a single program. Once a lot of the accessory stuff is deduplicated, each generator tends to become a very small function.

### CODE REVIEW

**Review by kris-watts-gravwell — CHANGES_REQUESTED** [review]
very nice.  I think we can delete the old ones and just have a "generator", no?

File: generators/unifiedGenerator/binary.ax [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Extraction defintion for the csv generator with everything
```
Comment (david-fritz-gravwell) [inline generators/unifiedGenerator/binary.ax:1]:
```suggestion
# Extraction defintion for the binary generator with everything
```

**Review by david-fritz-gravwell — CHANGES_REQUESTED** [review]

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Review by david-fritz-gravwell — APPROVED** [review]
Lgtm

**Pull request merged**